editor/animation_track_editor.cpp

```diff
@@ -4989,7 +4987,8 @@ void AnimationTrackEditor::_update_step(double p_new_step) {
 
 	EditorUndoRedoManager *undo_redo = EditorUndoRedoManager::get_singleton();
 	undo_redo->create_action(TTR("Change Animation Step"));
-	float step_value = p_new_step;
+	// To ensure that the conversion results are consistent between serialization and load, the value is snapped with 0.0625 to be a rational number.
+	float step_value = Math::round(p_new_step * FACTOR) / FACTOR;
```

Comment:
```suggestion
	float step_value = Math::min(9999.0f, Math::snapped(p_new_step, FACTOR));
```

Reply:
Only other question for that line is whether it might make sense to also add a comment noting that not returning the min value can break things? I will resolve this now and commit adding a comment if necessary after